.gitignore

```diff
@@ -348,3 +350,5 @@ MigrationBackup/
 
 # Ionide (cross platform F# VS Code tools) working folder
 .ionide/
+
+Fakebook.Profile/Fakebook.Profile.DataAccess/DML.sql
```

Comment:
i don't see the point of this

Reply:
Originally, it was added because I was avoiding any SQL/PostGreSQL-specific items until we knew what we were using (much earlier on), but this is just residual. It's been removed, though.